apps/barista/src/pages/single-page/single-page.ts

```diff
@@ -39,5 +40,14 @@ export class BaSinglePage implements BaPage, AfterViewInit {
 
   ngAfterViewInit(): void {
     this._recentlyOrderedService.saveToLocalStorage(this.contents);
+
+    const allTables = Array.prototype.slice.call(
+      document.querySelectorAll('table'),
+    );
```

Comment:
No need to array.slice.call, we do no longer support old IE versions, that were not able to iterate over nodeLists. 
We can switch that one to: 
```ts
const allTables = document.querySelectorAll('table');
```